contract_reader.py

```diff
@@ -112,6 +117,7 @@ def get_orderbook():
     current_user = '0x0000000000000000000000000000000000000000'
     current_offset = 0
     last_page_size = page_size
+    contract = init_contract(network)
```

Comment:
It looks like `get_orderbook`/`get_account_balances` could maybe be a method on a Contract class? You are instantiating the instance inside, the method, which seems like it should maybe be self instead?